compaction/compaction_strategy.cc

```diff
@@ -510,6 +510,17 @@ leveled_compaction_strategy::leveled_compaction_strategy(const std::map<sstring,
 {
 }
 
+void leveled_compaction_strategy::validate_options(const std::map<sstring, sstring>& options, std::map<sstring, sstring>& unchecked_options) {
+    size_tiered_compaction_strategy_options::validate(options, unchecked_options);
+
+    auto tmp_value = compaction_strategy_impl::get_value(options, SSTABLE_SIZE_OPTION);
```

Comment:
Even in this point in the patch series I still don't understand why we have two maps, "options" and "unchecked_options", and not just one map (or at worst, a map and a set, but I'm not sure even that is needed).